java/system-test/src/main/java/sleeper/systemtest/compaction/CompactionJobStatusJson.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.systemtest.compaction;
+
+import com.google.gson.Gson;
+import sleeper.compaction.job.status.CompactionJobStatus;
+import sleeper.core.record.process.status.ProcessRun;
+import sleeper.util.GsonConfig;
+
+import java.time.Duration;
+import java.time.Instant;
+
+public class CompactionJobStatusJson {
```

Comment:
Is this class needed? It's used in `isCompactionJobsFinished` in the logger, but this could be done by a helper method.

Reply:
It's the model for the JSON format. Gson uses reflection to convert to JSON. I didn't want to output every field as it'd make the logs very noisy.

Reply:
That makes sense, the spotbugs suppression you added makes this code a lot tidier